pkg/app/piped/executor/ecs/rollback.go

```diff
@@ -158,10 +159,17 @@ func rollback(ctx context.Context, in *executor.Input, platformProviderName stri
 			return false
 		}
 
-		if err := client.ModifyListeners(ctx, currListenerArns, routingTrafficCfg); err != nil {
+		modifiedRules, err := client.ModifyListeners(ctx, currListenerArns, routingTrafficCfg);
+		if err != nil {
 			in.LogPersister.Errorf("Failed to routing traffic to PRIMARY/CANARY variants: %v", err)
 			return false
 		}
+		
+		if len(modifiedRules) > 0 {
+			in.LogPersister.Infof("Successfully modified ELB listener rules: %s", strings.Join(modifiedRules, ", "))
```

Comment:
[nit] readable when n of rules >1 (ecs.go too)

```suggestion
			in.LogPersister.Infof("Successfully modified %d ELB listener rules: %s", len(modifiedRules), strings.Join(modifiedRules, ", "))
```